### DISCUSSION TITLE
Web UI does not show data despite it being present in kafka/clickhouse

### DISCUSSION THREAD

**Discussion by eahlys** [akvorado/akvorado#545] [Q&A]
Hi,

I have an issue with the Web UI (Akvorado 1.8.1). 
After docker-compose up, it works for a few minutes and then the "Visualize" tab gets blank and the max available timestamp in the graph is "now - 1h" (I cannot here go fruther than 10am)

![image](https://user-images.githubusercontent.com/7271127/223082555-a18b88a3-45e1-4cf3-9283-77eee7ff7cb3.png)

![image](https://user-images.githubusercontent.com/7271127/223078751-35baff59-cf1a-4828-ab24-c8aa76fc7b05.png)

**When I select "Last hour" as a timerange preset, everyting is fine. It's only when using a greater interval.**

However, inlet service in receiving flows, kafka is working as expected and clickhouse receives them correctly (they are present in flow database and the "clickhouse lag" is less than 2 second).

akvorado.yaml clickhouse config (only showing non-default config) :
```
  resolutions:
    - interval: 0
      ttl: 168h
    - interval: 5m
      ttl: 720h
    - interval: 1h
      ttl: 0
```

I don't know where to start to debug this, has anyone there experienced the same issue and solved id ?

**Comment by vincentbernat** [comment]
I suppose it gets confused with the TTL 0 for 1h.

**Comment by cristianmc** [comment]
If your interfaces are not correctly defined, by default you will not display any data with the filter inifboundary=external, try changing it to internal
